loginfo.py

```diff
@@ -0,0 +1,17 @@
+__author__ = "Keenan Manpearl"
+__date__ = "2023/03/01"
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+loss = []
+with open("loginfo.txt", "r") as file_handle:
+    for line in file_handle:
+        line = line.strip()
+        field = line.split()
+        if field[-1] != "nan":
+            loss.append(float(field[-1]))
+
+plt.plot(loss[::30])
+plt.ylim(70, 200)
+plt.show()
```

Comment:
Consider saving this plot as a png so others viewing the repo can quickly see loss results.